olclient/schemata/work.schema.json

```diff
@@ -59,10 +66,18 @@
       "required": [ "type", "author" ],
       "properties": {
         "type": {
-          "type": "string",
-          "enum": ["/type/author_role"]
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["key"],
+          "properties": {
+            "key": {
+              "enum": ["/type/author_role"]
+            }
+          }
         },
-        "author": { "$ref": "#/definitions/author" }
+        "author": { "$ref": "#/definitions/author" },
+        "role":   { "type": "string" },
+        "as":     { "type": "string" }
```

Comment:
Ooooh! I've never seen `as` before, do you have an example? Is it for pseudonyms?

Reply:
@cdrini Unfortunately I don't think `as` is used at all in the data, but it appears to be part of the intended design as documented https://openlibrary.org/type/author_role

`role` is only used a couple of times in the data, and currently the UI has nothing to display it, but I would love to add something to show editor, translator, 'introduction by' or whatever else. 

My interpretation  of `as` is for aliases or pseudonyms too, which would be great to capture and display.

The `author_role` property makes sense only with these extra fields though, otherwise a work would just contain a list of `authors`. I had been wondering why works had this extra level of data to specify author. We should take advantage of it!

There is a separate conversation to have about where `author_roles` should live. Currently `author_roles` is only on works. Editions have an optional plain `authors` list, but the UI ignores edition authors and only displays work authors.

I would suggest it is appropriate to have `author_roles` at an edition level too since translators only apply to the translated edition, editors can change over time, and other contributors can be involved in producing a particular edition. An edition should display a combined work + edition `author_roles` list of unique author+role combinations. Again, probably a bigger discussion to be had later! :)